backend/src/models/Post.js

```diff
@@ -4,6 +4,12 @@ export default {
   id: { type: 'string', primary: true, default: uuid },
   activityId: { type: 'string', allow: [null] },
   objectId: { type: 'string', allow: [null] },
+  image: {
+    type: 'relationship',
+    relationship: 'HERO_IMAGE',
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-18T15:57:22Z" title="Tuesday, February 18th 2020, 4:57:22 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,6 +4,12 @@ export default {
   id: { type: 'string', primary: true, default: uuid },
   activityId: { type: 'string', allow: [null] },
   objectId: { type: 'string', allow: [null] },
+  image: {
+    type: 'relationship',
+    relationship: 'TEASER_IMAGE',
```


Since this is a good chance to rename things... :wink: why is the post image even called `TEASER_IMAGE`? In my understanding (and according to the first google result) a teaser is "a short introductory advertisement for a product that stimulates interest by remaining cryptic". Not really what the post image is doing, right? At least not anymore.

In the new `BaseCard` component I called it `HeroImage`, which is – I admit – a very techy term but at least semantically closer to what it's doing: "hero image is often the first visual element a visitor encounters on the site". Well, in our case the post. Would love to hear your thoughts, @mattwr18!

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-19T08:43:26Z" title="Wednesday, February 19th 2020, 9:43:26 am +01:00">Feb 19, 2020</time>_

---


It is called `TeaserImage` because that is what it was called in the old `alpha` system, and I copied the naming convention without thinking about whether it should be renamed. 
It does however kinda make sense to me cause the image teases what the article/post is about, at least that is the way some use the image. 
Having said that, I have no issue renaming it, and `HeroImage` would be a good choice, in my mind.